app/src/components/buttons/multiActionButton/multiActionButton.scss

```diff
@@ -0,0 +1,148 @@
+.multi-action-button {
+  display: flex;
+  align-items: center;
+  position: relative;
```

Comment:
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
Fixed